src/solidity-frontend/solidity_grammar.h

```diff
@@ -221,7 +221,9 @@ enum StatementT
   ForStatement,          // rule for-statement
   IfStatement,           // rule if-statement
   WhileStatement,
-  StatementTError
+  StatementTError,
```

Comment:
What does the `StatementTError` do?

Reply:
Unrelated to this PR, but the only place it's being used is in `get_statement_t()` for unhandled statements.

Reply:
It can probably go away since the place where it's being used is unreachable anyway (protected by `abort()`).

Reply:
@JacobYiu  May I ask if you can resolve this issue?

Reply:
Yes. Will Lint and Push Once I can get the regression tests passing